### PR TITLE
add entityinfo、propertyinfo document,

### PR DESCRIPTION
**Pull request by adeewu**

### CODE REVIEW

**Comment by wakuflair** [comment]
It's better to directly copy the text of Xml node(`SingleLineDocumentationCommentTrivia`) to the generated code, 
otherwise you may need to handle some special formats, like one line comment: `/// <summary>property</summary>`.

**Comment by adeewu** [comment]
@wakuflair I have resubmitted to process as you like

File: src/AbpHelper.Core/Steps/Abp/EntityParserStep.cs [hunk 1]
```diff
@@ -123,5 +128,42 @@ protected override async Task<ActivityExecutionResult> OnExecuteAsync(WorkflowEx
...
+                .Where(p => p.Name.ToString() == "Description" || p.Name.ToString() == "DescriptionAttribute")
+                .Select(p => p.ArgumentList?.Arguments.Select(a => a.ToString().Trim('"')).JoinAsString(";"))
+                .JoinAsString(";");
+        }
+
+        private List<SyntaxKind> documentKinds = new List<SyntaxKind>
+        {
+            //SyntaxKind.DocumentationCommentExteriorTrivia,
+            //SyntaxKind.EndOfDocumentationCommentToken,
+            //SyntaxKind.MultiLineDocumentationCommentTrivia,
```
Comment (wakuflair) [inline src/AbpHelper.Core/Steps/Abp/EntityParserStep.cs:145]: If these 3 lines are not used, better to remove them.

File: src/AbpHelper.Core/Steps/Abp/EntityParserStep.cs [hunk 2]
```diff
@@ -123,5 +128,42 @@ protected override async Task<ActivityExecutionResult> OnExecuteAsync(WorkflowEx
...
+                                    .Replace("<summary>", string.Empty)
+                                    .Replace("/// </summary>\r\n", string.Empty)
+                                    .Replace("/// </summary>", string.Empty)
+                                    .Replace("</summary>\r\n", string.Empty)
+                                    .Replace("</summary>", string.Empty)
+                                    .Trim()
+                                    .Replace("/// ", string.Empty)
+                                    .Replace("///", string.Empty)
+                                    .Replace("\r\n", string.Empty))
+                      .JoinAsString(";");
```
Comment (wakuflair) [inline src/AbpHelper.Core/Steps/Abp/EntityParserStep.cs:166]: Can we just use something like `SingleLineDocumentationCommentTrivia.GetText()` here?

**Comment by wakuflair** [comment]
Thanks!

**Pull request merged**